bee-message/bee-byte-cost/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "bee-byte-cost"
```

Comment:
I am not sure at all, but I think since this is the new dust protection crate, and "byte cost" is just a specific impl. of a dust protection mechanism, we may better call it `bee-dust-protection` as this is also a bit more descriptive IMO.

Reply:
Hmm, I agree these two concepts are often used interchangeably. I don't have anything agains renaming it, other than having to do the work :trollface:. Any thoughts @thibault-martinez ?

Reply:
It is true that the dust protection is currently based on the byte cost, but changing it would be breaking change anyway so it's not like we would benefit from a "dynamic" name. Also, AFAIK, the byte cost crate in itself only really computes the byte cost, which is on its own *not* a dust protection. Dust protection is the syntactical comparison of the amount against the byte cost and that can only happen within the bee-message crate.

Reply:
makes sense